vdpwi/model.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, config):
         def make_conv(n_in, n_out):
             conv = nn.Conv2d(n_in, n_out, 3, padding=1)
             conv.bias.data.zero_()
-            nn.init.xavier_normal(conv.weight)
+            nn.init.xavier_normal_(conv.weight)
```

Comment:
@daemon should be aware of this change. Does this mean we were not using `xavier_normal` initialization before since we are not setting the copy of the tensor to anything? If so this may affect the hyperparameter optimization and experimentation that was worked on.

Reply:
https://pytorch.org/docs/stable/_modules/torch/nn/init.html

According to their doc, this is the correct usage?

Reply:
In fact, it throws an warning and ask me to replace `xavier_normal` with `xavier_normal_`. That's why I make this change.

Reply:
Fortunately, this does not affect hyperparameter optimization and experimentation. In < 0.4.0, `xavier_normal` was an in-place operation. In 0.4.0+, they deprecated `xavier_normal` and changed the name to `xavier_normal_`, which better reflects PyTorch convention of having underscore suffixes for in-place operations.